pages/common/column.md

```diff
@@ -0,0 +1,19 @@
+# column
+
+> Format standard or file input into multiple columns, rows are filled before columns.
```

Comment:
I'd use "Format standard input or file", or "Format file or standard input". Either is clearer and don't require the reader to parsed the whole expression before extracting its meaning.